README.markdown

```diff
@@ -7074,6 +7074,24 @@ An optional Lua table can be specified as the last argument to this method to sp
 * `pool`
 	specify a custom name for the connection pool being used. If omitted, then the connection pool name will be generated from the string template `"<host>:<port>"` or `"<unix-socket-path>"`.
 
+* `pool_size`
+	specify the size of connection pool being used. Note that the size of the connection pool cannot be changed once the pool is created. If omitted,
+then the default setting in the [lua_socket_pool_size](#lua_socket_pool_size) config directive will be used.
+When the connection pool exceeds the available size limit, the least recently used (idle) connection already in the pool will be closed to make room for the current connection.
+Note that the cosocket connection pool is per Nginx worker process rather than per Nginx server instance, so the size limit specified here also applies to every single Nginx worker process.
+If the `backlog` is not specified, then the size of connection pool is only
+for idle connections. If you need to restrict the number of active connections in the pool, specify the `backlog` as well.
+This option was first introduced in the `dev` release.
```

Comment:
We need to add one level of indentation to these paragraphs otherwise they would break the current bullet list.